Launcher/Program.cs

```diff
@@ -69,6 +70,22 @@ static void Main(string[] args)
             string assemblyPath;
             job = leanEngineSystemHandlers.JobQueue.NextJob(out assemblyPath);
 
+            // Activate our PythonVirtualEnvironment; Cloud case we know we deploy to "/venv"
+            if (!string.IsNullOrEmpty(job.PythonVirtualEnvironment))
+            {
+                var path = "/venv";
+                Log.Trace($"Engine.Main(): Activating Python Virtual Environment at {path}");
+                PythonInitializer.ActivatePythonVirtualEnvironment(path);
+            }
+
+            // Activate our PythonVirtualEnvironment if defined in config
+            var configPythonVenv = Config.Get("python-venv");
+            if(!string.IsNullOrEmpty(configPythonVenv))
```

Comment:
nit - this should probably be an else? Maybe `job.PythonVirtualEnvironment` should just be the default value of `Config.Get("python-venv")` ?

Reply:
These guys are separate because of their use case.

The job.PythonVirtualEnvironment is really just for the cloud to flag that there is a env mounted. No matter what value it passes in, we know that the path in deployment will be "/venv".

Where as config value `python-venv` is for the local user, whos folder could live anywhere and they can define it in their config.json.